setup.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import os
+from os.path import join
+import setuptools
+
+
+setuptools.setup(
+    name='cpg-pipes',
+    version='0.3.0',
+    description='Hail Batch bioinformatics pipelines',
+    long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
+    url='https://github.com/populationgenomics/production-pipelines',
+    license='MIT',
+    packages=['cpg_pipes'],
+    package_data={'cpg_pipes': ['filter_cutoffs.yaml']},
+    include_package_data=True,
+    zip_safe=False,
+    scripts=[
+        join('scripts', fname) 
+        for fname in os.listdir('scripts') if fname.endswith('.py')
+    ],
+    install_requires=[
+        'click',
+        'pandas',
+        'hail>=0.2.90',
+        'cpg-gnomad',   # github.com/populationgenomics/gnomad_methods
+        'google-cloud-storage',
+        'fsspec',
+        'sample-metadata',
+        'analysis-runner',
+        'cloudpathlib[gs]',
+        'coloredlogs',
+        'types-PyYAML',  # https://mypy.readthedocs.io/en/stable/getting_started.html#library-stubs-and-typeshed
+    ],
+    keywords='bioinformatics',
+    classifiers=[
+        'Environment :: Console',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
```

Comment:
This should be MIT.

Did we accidentally specify GPLv3 somewhere else in our repos?

Reply:
Fixed!